Content.Server/Explosion/EntitySystems/ExplosionSystem.Processing.cs

```diff
@@ -464,7 +464,7 @@ private void ProcessEntity(
             }
         }
 
-        // ignite
+        // ignite components
```

Comment:
It's not igniting components, it's igniting entities *that have* a flammableComponent.